web/modules/custom/server_style_guide/css/server_style_guide.toolbar.css

```diff
@@ -7,3 +7,78 @@
 .toolbar-icon-style-guide::before {
   background-image: url(../assets/style_guide.svg);
 }
+.rounded-full {
```

Comment:
Why are all the classes needed? They shouldn't be - please submit this without

Reply:
Facing issue of when trying to compile css using below command:
npx tailwindcss -i ./src/pcss/style.pcss -o ./dist/css/style.css --watch

Rebuilding...
/var/www/drupal-starter/web/themes/custom/server_theme/node_modules/postcss-selector-parser/dist/parser.js:174
        return new Error(message);
               ^

Error: Unexpected '/'. Escaping special characters with \ may help.
    at Root._error (/var/www/drupal-starter/web/themes/custom/server_theme/node_modules/postcss-selector-parser/dist/parser.js:174:16)

Reply:
Please read the README of the Drupal-starter project

Reply:
yeah, it is working now.
ddev theme:watch